scripts/variantstore/wdl/GvsAssignIds.wdl

```diff
@@ -128,8 +128,8 @@ task AssignIds {
   String bq_labels = "--label service:gvs --label team:variants --label managedby:assign_ids"
 
   command <<<
-    set -e
-    set -x
+    PS4='\D{+%F %T} \w $ '
```

Comment:
nit: might want to add the comment `# Prepend date, time and pwd to xtrace log entries.` like you did elsewhere for clarity